internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -180,6 +180,45 @@ func resourceMsSqlVirtualMachine() *pluginsdk.Resource {
 					},
 				},
 			},
+			
+			"assessment_settings": {
+				Type:     pluginsdk.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"schedule": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &pluginsdk.Resource{
+								"weekly_interval": {
+									Type:     pluginsdk.TypeInt,
+									Optional: true,
+								},
+								"monthly_occurrence ": {
+									Type:     pluginsdk.TypeInt,
+									Optional: true,
+								},
+								"day_of_week  ": {
+									Type:     pluginsdk.TypeInt,
+									Optional: true,
+								},
+								"start_time  ": {
```

Comment:
there is extra space in `"monthly_occurrence "`, `"day_of_week  "` and `"start_time  "`

Reply:
@myc2h6o  thanks this should be resolved